tools/autoformat.sh

```diff
@@ -40,6 +40,10 @@ dotnet format whitespace "$SRC_DIR/tests/xtro-sharpie/xtro-report/xtro-report.cs
 dotnet format whitespace "$SRC_DIR/tests/xtro-sharpie/xtro-sanity/xtro-sanity.csproj"
 dotnet format whitespace --folder "$SRC_DIR"
 
+for file in "$SRC_DIR"/dotnet/Templates/Microsoft.*.Templates/*/*/.template.config/localize/*.json "$SRC_DIR"/dotnet/Templates/Microsoft.*.Templates/*/.template.config/localize/*.json; do
```

Comment:
I wonder if we should have this as a variable so it can be augmented when we have new localization file paths that we want to add else this line will get looooooong.

Reply:
Yes, it might be long, but imho we can wait until it becomes a problem before we spend time on fixing it :)